src/contracts/examples/ExampleContract.sol

```diff
@@ -0,0 +1,79 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.6.8;
+pragma experimental ABIEncoderV2;
+
+//import "hardhat/console.sol";
+
+import "../lib/ValueDecoder.sol";
+
+// contract with examples for working with signed integers in solidity
```

Comment:
the goal is not to show how to work with signet integers, but to show how to use new added method to `ValueDecoder` in scope of our oracle.

Reply:
sorry i misunderstood:
'add examples to solidity (in SDK) for how to use int'
- i can remove these functions from the example contract and test cases

-and the example contract can be used to provide a means to test the new ValueDecoder method?